osidb/serializer.py

```diff
@@ -919,19 +919,40 @@ class Meta:
         abstract = True
 
 
-class AffectCVSSSerializer(
+class AbstractCVSSSerializer(
     ACLMixinSerializer,
     AlertMixinSerializer,
     BugzillaSyncMixinSerializer,
     IncludeExcludeFieldsMixin,
     TrackingMixinSerializer,
+):
+    """Abstract serializer for FlawCVSS and AffectCVSS serializer"""
+
+    cvss_version = serializers.ChoiceField(
+        choices=FlawCVSS.CVSSVersion, source="version"
+    )
+    score = serializers.FloatField(read_only=True)
+    comment = serializers.CharField(required=False, allow_null=True, allow_blank=True)
+
+    class Meta:
+        abstract = True
+        fields = (
```

Comment:
In both `FlawCVSSSerilizer` and `AffectCVSSSerializer` you can now do:

```python
class Meta:
    ...
    fields = AbstractCVSSSeralizer.Meta.fields + ["flaw"]

class Meta:
    ...
    fields = AbstractCVSSSeralizer.Meta.fields + ["affect"]
```

Reply:
Thanks, fixed. Moved the `[]` to the beginning to make the generated schema a bit clearer because these are IMO one of the most important fields. Maybe it's just a cosmetic preference.